build/scripts/PullRequestValidation/ValidateIssuesForPullRequest.ps1

```diff
@@ -0,0 +1,47 @@
+using module ..\GitHub\GitHubPullRequest.class.psm1
+using module ..\GitHub\GitHubIssue.class.psm1
+
+param(
+    [Parameter(Mandatory = $true)]
+    [string] $PullRequestNumber,
+    [Parameter(Mandatory = $true)]
+    [string] $Repository
+    )
+
+# Set error action
+$ErrorActionPreference = "Stop"
+
+Write-Host "Validating PR $PullRequestNumber"
+
+$pullRequest = [GitHubPullRequest]::Get($PullRequestNumber, $Repository)
+$prDescription = $pullRequest.GetBody()
+$issuesStartingPoint = "Fixes #"
```

Comment:
Fixes is only one of the keywords you can use (https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword). Why not set issuesStartingPoint to be "# Work Items"?  

E.g. 
$workItemsStartingIndex = $description.IndexOf("# Work Item(s)")

Reply:
If it's a whole section, it might be hard to determine where it ends and what exactly to scan for.

I say we scan for whatever we put in the PR template. What do you think?